src/AM.Condo/Targets/Initialize.targets

```diff
@@ -82,6 +82,9 @@
   <PropertyGroup>
     <RepositoryProvider Condition=" '$(RepositoryProvider)' == '' AND Exists('$(RepositoryRoot)$(slash).git') ">git</RepositoryProvider>
 
+    <IsPullRequest Condition=" '$(PullRequestId)' != '' ">true</IsPullRequest>
```

Comment:
How difficult would it be to write a unit test for this change?  I'm not very familiar with condo yet, so I'm not sure if that would require a bunch of setup or not.

Reply:
Impossible. Would require an integration test of sorts. It’s on the backlog. We rely on MSBuild for many things, which isn’t testable.